zuul.d/projects.yaml

```diff
@@ -21,7 +21,7 @@
       - ansible-collections-arista-eos
       - ansible-collections-cisco-ios
       - ansible-collections-cisco-iosxr-netconf
-      - ansible-collections-juniper-junos
+      - ansible-collections-juniper-junos-netcommon-jobs
```

Comment:
Looks like it should be `ansible-collections-juniper-junos-netcommon` without the `-jobs` suffix.